bin/phoenix_utils.py

```diff
@@ -77,7 +77,7 @@ def findClasspath(command_name):
     return tryDecode(subprocess.Popen(command, shell=True, stdout=subprocess.PIPE).stdout.read())
 
 def setPath():
-    PHOENIX_CLIENT_JAR_PATTERN = "phoenix-client.jar"
+    PHOENIX_CLIENT_JAR_PATTERN = "phoenix-client*.jar"
```

Comment:
There's a chance that this will find the -sources.jar
Modifying the glob to be similar to PHOENIX_LOADBALANCER_JAR_PATTERN should solve that
`-*[!s].jar` should filter both -sources.jar, and -tests.jar (even though we don't have a -tests.jar now)